contracts/seed/Seed.sol

```diff
@@ -242,13 +239,14 @@ contract Seed {
         require(tokenLocks[msg.sender].fundingAmount > 0, "Seed: zero funding amount");
         Lock storage tokenLock = tokenLocks[msg.sender];
```

Comment:
Actually aren't we making the tx gas price a bit higher using storage variable here?

Reply:
I though so too, but it was like this so I didn't

Reply:
I'll create issue for this one